tox.ini

```diff
@@ -10,6 +10,9 @@ deps =
 passenv =
     KUBECONFIG
 
+[testenv:unit]
+commands = pytest -v {toxinidir}/tests/unit
```

Comment:
consider reporting coverage at this point.  If you've got unit tests you can get the coverage with `pytest-cov` and using `--cov-report=term-missing` as an argument to get nice reports of what isn't unit tested -- even block a PR in the event the coverage isn't high enought

Reply:
good suggestion.  I dont think we've done coverage on any of our charm repos yet

Reply:
+1 I would like us to add coverage at some point.

Reply:
Added, thanks for the suggestion @addyess.